src/VirtoCommerce.CustomerModule.Data/Search/Indexing/MemberDocumentBuilder.cs

```diff
@@ -27,22 +27,19 @@ public MemberDocumentBuilder(IMemberService memberService, IDynamicPropertySearc
         public virtual async Task<IList<IndexDocument>> GetDocumentsAsync(IList<string> documentIds)
         {
             var members = await GetMembers(documentIds);
-            var result = new List<IndexDocument>();
-
-            foreach (var member in members)
-            {
-                result.Add(await CreateDocument(member));
-            }
 
+            IList<IndexDocument> result = members.Select(CreateDocument).ToArray();
             return result;
         }
 
-        protected virtual Task<Member[]> GetMembers(IList<string> documentIds)
+        protected virtual async Task<Member[]> GetMembers(IList<string> documentIds)
```

Comment:
You trying to fix one breaking change, but introduce another

Reply:
Are you sure? please read the answer https://stackoverflow.com/a/44397025/4583735